app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -175,7 +175,7 @@ export default function Navigation() {
   }
   return (
     <AppBar
-      position="fixed"
+      position="sticky"
```

Comment:
I wonder if this is why the report a bug button now triggers an annoying horizontal scroll bar. At least it's better than before where it was cut off? 🤷 

Might be worth making a separate bug issue to fix than polluting this PR more.

Reply:
I introduced this here in order to avoid keep having the custom paddings. This shouldn't affect how the scrollbars displays in itself (in this context, sticky is just a fixed position that "makes room for itself"